signer/src/storage/postgres.rs

```diff
@@ -329,6 +329,9 @@ impl super::DbRead for PgStore {
         context_window: u16,
         threshold: u16,
     ) -> Result<Vec<model::DepositRequest>, Self::Error> {
+        // TODO: Make sure we get only pending deposits, don't include ones
```

Comment:
Should there be a tracking issue for this?

Reply:
Oh, yes. Will update.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/540/commits/16c7b7d5b5c8ecab23c0bb435c0d02933c204de4. I opened https://github.com/stacks-network/sbtc/issues/543 for this.